### PR TITLE
Fallback to fetch v12 (#10)

### PR DESCRIPTION
**Pull request by anchitj**
If any toppars have a zero topic id

### How to Reproduce
Set inter.broker.protocol.version to 2.4 with 3.7 and fetch would fail always.

### CODE REVIEW

File: src/rdkafka_fetcher.c [hunk 1]
```diff
@@ -943,7 +943,21 @@ static void rd_kafka_broker_fetch_reply(rd_kafka_t *rk,
         }
 }
 
+/**
+ * @brief Check if any toppars have a non-zero topic id.
```
Comment (anchitj) [inline src/rdkafka_fetcher.c:947]: Will change to `zero topic id`

File: src/rdkafka_fetcher.c [hunk 2]
```diff
@@ -979,7 +993,12 @@ int rd_kafka_broker_fetch_toppars(rd_kafka_broker_t *rkb, rd_ts_t now) {
 
         ApiVersion = rd_kafka_broker_ApiVersion_supported(rkb, RD_KAFKAP_Fetch,
                                                           0, 16, NULL);
-        rkbuf      = rd_kafka_buf_new_flexver_request(
+
+        /* Fallback to version 12 if topic id is null which can happen if
+         * inter.broker.protocol.version is old */
```
Comment (anchitj) [inline src/rdkafka_fetcher.c:998]: Change old to <2.8

**Review by emasab — CHANGES_REQUESTED** [review]
Thanks for the fix! Let's make your changes and my comment and rebase it, then I'm approving

**Review by emasab — APPROVED** [review]
LGTM Thanks Anchit

**Pull request merged**